packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -137,7 +159,7 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 									) }
 								></div>
 							) }
-							{ ! completed && (
+							{ ! completed && showActionButton && (
```

Comment:
Somewhat related to this PR, but probably can be handled in a follow-up:  when a list item is open, it still appears clickable via the cursor and is also outlined, even though the action button is the only clickable element.

Also wondering if the time text could use just a bit of spacing.  cc @elizaan36 

<img width="703" alt="Screen Shot 2021-07-06 at 11 27 44 AM" src="https://user-images.githubusercontent.com/10561050/124628548-93000700-de4e-11eb-86f9-c1989d970fc0.png">

Reply:
I can update this as part of this PR, this is probably a change in the actual task-list styling versus the task-item component styling (or at-least it should be).

Reply:
Fixed the cursor and outline in this commit: https://github.com/woocommerce/woocommerce-admin/pull/7263/commits/20d0ab6a20b151c5ffcf657ff4ca763b685a97bd

Reply:
Awesome 🎉  Looks great.

Reply:
The time estimate still seems very close (unrelated as its shown without the collapsible list too) but we can wait for @elizaan36 to chime in.

<img width="709" alt="Screen Shot 2021-07-06 at 3 30 51 PM" src="https://user-images.githubusercontent.com/10561050/124656692-84761780-de6f-11eb-84f2-cae14cfbd47c.png">

Reply:
@joshuatf this is actually an issue with the above changes, and the maxHeight it sets on the expandable item. It doesn't take margin in effect, which makes the expanded area cropped a tiny bit (causing the minutes to be very close).